chaste_codegen/data/tests/cellml/test_V_not_state_mparam.cellml

```diff
@@ -44,7 +44,7 @@
         </math>
     </component>
     <component name="membrane">
-        <variable name="time" public_interface="in" units="second"/>
```

Comment:
Why this change?

Reply:
when I rant these files through pycml it complained that this couldn't be public interface in, but was happy when I left it alltogether, also seemed to work for codegen